mflowgen/glb_top/construct.py

```diff
@@ -215,12 +215,17 @@ def construct():
   #-----------------------------------------------------------------------
 
   g.update_params( parameters )
-  # Since we are adding an additional input to the init node, we must add
-  # that input to the order parameter for that node, so it actually gets run
-  init.update_params(
-                     {'order': "\"main.tcl quality-of-life.tcl floorplan.tcl add-endcaps-welltaps.tcl "\
-                               "pin-assignments.tcl make-path-groups.tcl dont_touch.tcl reporting.tcl\""}
```

Comment:
Didn't look at this closely enough... the dont_touch was left out of this PR. creating a fix now